src/batou/component.py

```diff
@@ -391,8 +391,8 @@ def __exit__(self, type, value, tb):
     def log(self, message, *args):
         """Log a message to console during deployment.
 
-        The message is %-substituted with *args, if it is put out. and prefixed
-        with the hostname automatically.
+        The message is %-substituted with *args*, if it is put out. and
```

Comment:
This breaks the semantics. "*args" has a literal meaning in Python. The asterisk is part of the message, not formattion. Not sure what the right quoting would be ...